object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,10 @@
+/* exported toObject */
+function toObject(keyValuePair) {
+  let object = {};
+  let objectTwo = '';
+  const i = 0;
+  object[i] = keyValuePair[i];
```

Comment:
I don't think this is doing anything - especially since you're overwriting it with a new object literal on line 9